cloudify_marketplace/provision/check_prerequisites

```diff
@@ -0,0 +1,6 @@
+#! /usr/bin/env bash
+MIN_RAM_GB=3
```

Comment:
Ah, I'll remove this now. It was primarily to avoid running through the entire bootstrap then failing, but as your checks are in now, the time cost is that of doing the initial downloads/checks (so a minute or so at most).